posts/urllib3-pipenv-standalone-test.meta

```diff
@@ -0,0 +1,4 @@
+.. title: Urllib3 Pipenv Standalone Pytest Report 
+.. slug: urllib3-pipenv
+.. date: 2019/06/16 16:00
+.. tags: compiler,Python,Nuitka
```

Comment:
There should be a tag gsoc or gsoc2019, use it please

Reply:
Also this hasn't really anything to do with pipenv yet, does it? I think virtualenv may be a better fit, but actually it's more a tool. I would suggest "Urllib3 wheel with Nuitka and test results"